pom.xml

```diff
@@ -808,5 +808,12 @@
                 <module>dev/kyuubi-tpcds</module>
             </modules>
         </profile>
+
+        <profile>
+            <id>spark-2.4</id>
+            <properties>
+                <spark.version>2.4.7</spark.version>
+            </properties>
```

Comment:
Should we also set value for hive.version here?

Reply:
https://github.com/apache/spark/blob/e0e1e21ee84bb3cb20eefee982da59afaa250a2c/pom.xml#L129-L131

Reply:
we dont rely on the builtin hive of spark，and it wont conflict  with that either